lib/agent.js

```diff
@@ -424,4 +430,12 @@ function urlPropertyToPacketOption(url, req, property, option, separator) {
       }))
 }
 
+Agent.prototype._convertMulticastToUnicastRequest = function (req, rsinfo) {
+  var reqOpts = {"host": rsinfo.address.split('%')[0], "port": rsinfo.port}
+  var unicast_req = this.request(reqOpts)
+  unicast_req._events.response = req._events.response
```

Comment:
Do not touch `_events`. Use public methods from `EventEmitter` to interact with it/

Reply:
Oh, I see, thank you. I will fix this.